[2023-10-30T06:31:44.952+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: opensky_data_processing.fetch_opensky_data scheduled__2023-10-29T15:20:00+00:00 [queued]>
[2023-10-30T06:31:45.008+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: opensky_data_processing.fetch_opensky_data scheduled__2023-10-29T15:20:00+00:00 [queued]>
[2023-10-30T06:31:45.008+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 101
[2023-10-30T06:31:45.107+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_opensky_data> on 2023-10-29 15:20:00+00:00
[2023-10-30T06:31:45.146+0000] {standard_task_runner.py:57} INFO - Started process 145144 to run task
[2023-10-30T06:31:45.183+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'opensky_data_processing', 'fetch_opensky_data', 'scheduled__2023-10-29T15:20:00+00:00', '--job-id', '1727', '--raw', '--subdir', 'DAGS_FOLDER/DataFetcher.py', '--cfg-path', '/tmp/tmpn5jozau7']
[2023-10-30T06:31:45.202+0000] {standard_task_runner.py:85} INFO - Job 1727: Subtask fetch_opensky_data
[2023-10-30T06:31:45.507+0000] {task_command.py:416} INFO - Running <TaskInstance: opensky_data_processing.fetch_opensky_data scheduled__2023-10-29T15:20:00+00:00 [running]> on host ip-172-31-33-29.ec2.internal
[2023-10-30T06:31:46.153+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='adayani' AIRFLOW_CTX_DAG_ID='opensky_data_processing' AIRFLOW_CTX_TASK_ID='fetch_opensky_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-29T15:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-29T15:20:00+00:00'
[2023-10-30T06:33:37.664+0000] {local_task_job_runner.py:274} WARNING - Recorded pid 147175 does not match the current pid 145144
[2023-10-30T06:33:37.735+0000] {process_utils.py:131} INFO - Sending 15 to group 145144. PIDs of all processes in the group: [145144]
[2023-10-30T06:33:37.735+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 145144
[2023-10-30T06:33:37.735+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-10-30T06:33:37.736+0000] {logging_mixin.py:151} INFO - Error: Task received SIGTERM signal
[2023-10-30T06:33:37.738+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-30T06:33:37.840+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=opensky_data_processing, task_id=fetch_opensky_data, execution_date=20231029T152000, start_date=20231030T063335, end_date=20231030T063337
[2023-10-30T06:33:37.963+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=145144, status='terminated', exitcode=0, started='06:31:44') (145144) terminated with exit code 0
